benchmarks/profiler_benchmark/profiler_bench.py

```diff
@@ -0,0 +1,24 @@
+import torch
+import time
+
+def run_profiler_benchmark(n_iters):
+    """
+    Run a tight loop with a single matmul under the autograd profiler
+    and report the latency. Useful for profiling changes to the profiler
+    that may affect its performance.
+    """
+    latencies = []
+    for i in range(n_iters):
+        start = time.time()
```

Comment:
btw no need to manually use timers, we can use python's timeit package

Reply:
Sounds good, I'll update the PR.